sdk/cosmosdb/cosmos/test/public/functional/item.spec.ts

```diff
@@ -860,12 +1976,32 @@ describe("patch operations", function () {
           value: 5,
         },
       ];
-      const { resource: addItem } = await container.item(addItemId).patch(operations);
+      const { resource: addItem } = await container
+        .item(addItemId, ["asdf", "dfs"])
+        .patch(operations);
       assert.strictEqual(addItem.num, 5);
       assert.strictEqual(addItem.existingObj.newKey, "newVal");
       assert.strictEqual(addItem.laster, "c");
       assert.strictEqual(addItem.last, "b");
       assert.strictEqual(addItem.removable, undefined);
+
+      const { resource: addItemWithOnePartitionKey } = await container
+        .item(addItemWithOnePartitionKeyId, ["sdfasdf", undefined])
+        .patch(operations);
+      assert.strictEqual(addItemWithOnePartitionKey.num, 5);
+      assert.strictEqual(addItemWithOnePartitionKey.existingObj.newKey, "newVal");
+      assert.strictEqual(addItemWithOnePartitionKey.laster, "c");
+      assert.strictEqual(addItemWithOnePartitionKey.last, "b");
+      assert.strictEqual(addItemWithOnePartitionKey.removable, undefined);
+
+      const { resource: addItemWithNoPartitionKey } = await container
+        .item(addItemWithNoPartitionKeyId, [undefined, undefined])
+        .patch(operations);
+      assert.strictEqual(addItemWithNoPartitionKey.num, 5);
+      assert.strictEqual(addItemWithNoPartitionKey.existingObj.newKey, "newVal");
+      assert.strictEqual(addItemWithNoPartitionKey.laster, "c");
+      assert.strictEqual(addItemWithNoPartitionKey.last, "b");
+      assert.strictEqual(addItemWithNoPartitionKey.removable, undefined);
```

Comment:
@v1k1 this file has grown too much, can we do some refactoring here?
Something like creating test objects in different file.
Clubbing partitioning cases together in a separate file.

Reply:
Will pick this up in future? This will require a lot of code rearranging.